starlark/profile_test.go

```diff
@@ -76,7 +77,7 @@ fibonacci(100000)
 			t.Errorf("output did not contain %q", want)
 		}
 	}
-	if t.Failed() {
+	if true || t.Failed() {
```

Comment:
Leftover from debugging?

Reply:
Indeed, thanks. But it's close to midnight on a Saturday, when all right-thinking people are out dancing. Step away your from keyboard, man.